.github/workflows/python-versions.yml

```diff
@@ -48,9 +45,9 @@ jobs:
           path: |
             ${{ steps.pip.outputs.pip-cache-dir }}
           key: |
-            ${{ matrix.os }}-py${{ matrix.python-version }}-pip${{ steps.pip.outputs.pip-version }}-${{ hashFiles('requirements-*.txt') }}
+            python-versions-${{ matrix.os }}-py${{ matrix.python-version }}-${{ hashFiles('requirements-*.txt') }}
```

Comment:
We only need to track the `pip` version for the dev env workflow.

What do you think of `python-deps` instead of `python-versions`?

Reply:
`python-deps` makes more sense imo